lib/datasets/pmjt-dataset-list.rb

```diff
@@ -0,0 +1,30 @@
+module Datasets
+  class PmjtDatasetList < Dataset
+    LATEST_VERSION = '201901'
+    URL_FORMAT = "http://codh.rois.ac.jp/pmjt/list/pmjt-dataset-list-%{version}.csv".freeze
+
+    def initialize(version: LATEST_VERSION)
+      super()
+      @metadata.id = "pmjt-dataset-list-#{version}"
+      @metadata.name = "PmjtDatasetList #{version}"
+      @metadata.url = URL_FORMAT % {version: version}
+      @metadata.licenses = ["CC-BY-SA-4.0"]
+      @metadata.description = "Japanese Classic Book #{version}"
+
+      @data_path = cache_dir_path + (@metadata.id + ".csv")
+    end
+
+    def each(&block)
+      return to_enum(__method__) unless block_given?
+
+      download(@data_path, @metadata.url) unless @data_path.exist?
+      CSV.open(@data_path, headers: :first_row, encoding: "Windows-31J:UTF-8") do |csv|
+        csv.each do |row|
+          record = row.to_h
+          record.transform_keys!(&:to_sym)
```

Comment:
It seems that all keys are Japanese. So `Symbol` isn't suitable for them.

How about creating a `Struct` for this record (we did this in other dataset such as `Datasets::Wikipedia`) and using English key instead of Japanese key? e.g.:

```ruby
Record = Struct.new(:unit,
                    :open_data_category,
                    ...)

# ...
Record.new(unit: row["(単位)"],
           open_data_category: row["オープンデータ分類"],
           ...)
# ...
```

Method access (`record.unit`) will be easier to use than array reference access (`record["(単位)"]`) for users.